Form-Controls/styles.css

```diff
@@ -0,0 +1,88 @@
+main {
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+}
+#form-info {
+  padding: 20px;
+  background-color: rgba(199, 197, 203, 0.44);
+  width: 500px;
+  height: auto;
+  border: 1px solid rgb(172, 170, 170);
+}
+
+#btn {
+  padding: 0.25em 0.75em;
+  min-width: 10ch;
+  min-height: 30px;
+  text-align: center;
+  line-height: 1.1;
+  background-color: rgb(163, 139, 159);
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: bold;
+  border-radius: 45%;
+  align-items: center;
+}
+
+#btn:hover {
+  text-shadow: 2px;
+  color: rgb(8, 19, 221);
+}
+.form-control {
+  font-size: 16px;
+  font-size: max(16px, 1em);
+  font-family: inherit;
+  padding: 0.25em 0.5em;
+
+  border: 2px solid var(--input-border);
+  border-radius: 4px;
+
+  /* cursor: not-allowed; */
+}
+.form-control:not(textarea) {
+  line-height: 1;
+  height: 2.25rem;
+}
+.form-control:focus {
+  border-color: hsl(
+    var(--input-focus-h),
```

Comment:
Where are these variables defined?

Reply:
I was checking something and forgot to delete it.